apps/public-reference/pages/listing.tsx

```diff
@@ -3,41 +3,87 @@ import ReactDOMServer from "react-dom/server"
 import Head from "next/head"
 import axios from "axios"
 import Markdown from "markdown-to-jsx"
-import { Listing } from "@bloom-housing/core"
+import { Listing, ListingEvent, ListingEventType } from "@bloom-housing/core"
 import {
   AdditionalFees,
   ApplicationSection,
   ApplicationStatus,
   BasicTable,
   Description,
   ExpandableText,
+  getOccupancyDescription,
   GroupedTable,
   GroupedTableGroup,
+  groupNonReservedAndReservedSummaries,
   Headers,
-  InfoCard,
   ImageCard,
+  imageUrlFromListing,
+  InfoCard,
   LeasingAgent,
-  ListingDetails,
   ListingDetailItem,
+  ListingDetails,
   ListingMap,
   ListSection,
   MetaTags,
+  occupancyTable,
   OneLineAddress,
   PreferencesList,
+  t,
   UnitTables,
   WhatToExpect,
-  imageUrlFromListing,
-  getOccupancyDescription,
-  groupNonReservedAndReservedSummaries,
-  occupancyTable,
-  t,
 } from "@bloom-housing/ui-components"
 import Layout from "../layouts/application"
+import moment from "moment"
 
 interface ListingProps {
   listing: Listing
 }
 
+function EventDateSection(props: { event: ListingEvent }) {
```

Comment:
@pbn4 To keep things consistent with other parts of the codebase, can we use the `const EventDateSection = (…) => {` syntax instead? (And similar for the next two functions)